oga/frontend/src/store/actions/questionActions.js

```diff
@@ -0,0 +1,29 @@
+import * as actionTypes from './actionTypes.js';
+import axios from 'axios';
+
+import { push } from 'connected-react-router';
+
+axios.defaults.xsrfCookieName="csrftoken";
+axios.defaults.xsrfHeaderName="X-CSRFTOKEN";
+
+export const createQuestion_ = (question) => {
+  return {
+    type: actionTypes.CREATE_QUESTION,
+    //id: question.id,
```

Comment:
We need id: question.id.

Reply:
I don't think we need a question id when creating it. It is automatically assigned by the backend.